results.js

```diff
@@ -309,7 +309,7 @@ document.addEventListener("DOMContentLoaded", () => {
         if (unpaired_intern_info !== undefined)
             {
                 let intern1 = added_intern;
-                let intern2 = remove_unpaired_intern(unpaired_intern_info[1]);
+                let intern2 = remove_unpaired_intern(unpairedInterns,unpaired_intern_info[1]);
```

Comment:
What was this for?

Reply:
Not passing in unpaired interns causes an error. This is part of Alan's code he said to add that because we need it. I guess he has added it multiple times but doesn't show up for others? so pushing this will fix it on main.